loadtest/store/memstore/random.go

```diff
@@ -12,9 +12,13 @@ import (
 )
 
 var (
-	ErrEmptyMap    = errors.New("memstore: cannot select from an empty map")
-	ErrEmptySlice  = errors.New("memstore: cannot select from an empty slice")
-	ErrLenMismatch = errors.New("memstore: cannot select from a map, not enough elements")
+	ErrEmptyMap       = errors.New("memstore: cannot select from an empty map")
+	ErrEmptySlice     = errors.New("memstore: cannot select from an empty slice")
+	ErrLenMismatch    = errors.New("memstore: cannot select from a map, not enough elements")
+	ErrTeamNotFound   = errors.New("memstore: team not found")
+	ErrUserNotSet     = errors.New("memstore: user is not set")
+	ErrNoTeamFound    = errors.New("memstore: no team found")
+	ErrNoChannelFound = errors.New("memstore: no channel found")
```

Comment:
We should be consistent - ErrChannelNotFound / memstore: channel not found.

Reply:
Same reasoning as above.